src/util/irep.h

```diff
@@ -121,9 +121,9 @@ class tree_nodet : public ref_count_ift<sharing>
   // These are not stable.
   typedef std::vector<treet> subt;
 
-// named_subt has to provide stable references; with C++11 we could
-// use std::forward_list or std::vector< unique_ptr<T> > to save
-// memory and increase efficiency.
+  // named_subt has to provide stable references; with C++11 we could
```

Comment:
While you're here: this comment is now out of date (we require C++11 unconditionally)